modules/oe_theme_helper/tests/src/Functional/Plugin/field_group/FieldInPageNavigationFormatterTest.php

```diff
@@ -0,0 +1,284 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_theme_helper\Functional\Plugin\field_group;
+
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\Tests\BrowserTestBase;
+use Drupal\Tests\field_group\Functional\FieldGroupTestTrait;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Test the in-page navigation field group formatter.
+ */
+class FieldInPageNavigationFormatterTest extends BrowserTestBase {
+
+  use FieldGroupTestTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'node',
+    'text',
+    'field_ui',
+    'field_group',
+    'oe_theme_helper',
+  ];
+
+  /**
+   * Fields configurations for testing.
+   *
+   * @var array
+   */
+  protected $fields;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+    // Enable oe_theme and set it as default.
+    $this->assertTrue($this->container->get('theme_installer')->install(['oe_theme']));
+    $this->container->get('config.factory')
+      ->getEditable('system.theme')
+      ->set('default', 'oe_theme')
+      ->save();
+
+    // Rebuild the ui_pattern definitions to collect the ones provided by
+    // oe_theme itself.
+    $this->container->get('plugin.manager.ui_patterns')->clearCachedDefinitions();
```

Comment:
We don't use patterns in this test, remove this.

Reply:
Removed